hasher-matcher-actioner/hmalib/lambdas/actions/action_evaluator.py

```diff
@@ -15,6 +17,10 @@
 from hmalib.lambdas.actions.action_performer import perform_label_action
 
 logger = get_logger(__name__)
+sqs_client = boto3.client("sqs")
+
+ACTIONS_QUEUE_URL = os.environ["ACTIONS_QUEUE_URL"]
+REACTIONS_QUEUE_URL = os.environ["REACTIONS_QUEUE_URL"]
```

Comment:
nit: David has advised me before that if we move these to a class it can be easier to mock in tests. See `FetcherConfig` class in `fetcher.py`